proto/viam/app/v1/app.proto

```diff
@@ -1162,6 +1165,13 @@ message DeleteKeyRequest {
 
 message DeleteKeyResponse {}
 
+message RenameKeyRequest {
+  string id = 1;
+  string token_name = 2;
+}
+
+message RenameKeyResponse {}
```

Comment:
can we parrot back the new name? in case we do any stripping out of special characters etc

Reply:
yup! great sanity check